### ISSUE TITLE
Name the export settings file WITH "OsmAnd" prefix

### ISSUE THREAD

**Issue by pstein** [osmandapp/OsmAnd#17191]
In the past I exported the settings from OsmAnd to a folder on my Android Smartphone.

I wanted to restore these settings now but didn't remember the exact file name of the settings file.

I expected that the settings file name is something like  "OsmAnd-settings_11-03-23.osf"

However it was a very tedious search.

At the end it turns out that OsmAnd named it 
"Export_11-03-23.osf"

This is a confusing, counterintuitive" file name. Approx 99% of all other Apps put a reference to the creating file.

So I would like to ask the OsmAnd developers to rename the settings file into a "normal" meaningful file name.
As a bonus you could add the OsmAnd version like in 

 "OsmAnd-v4.4.7-settings_11-03-23.osf"

Can you change this in the next release?
Thank you

**Comment by pebogufi** [comment]
Good idea, but please use iso date yyyy-mm-dd for better sorting.

**Comment by Rowin63** [comment]
Like mentioned in an other issue (don't want to search, but it exists) already!

**Comment by pebogufi** [comment]
> Similar to this, mentioned already almost a year ago in #14830!

Yes, date format is currently ok, but the proposal above uses the unwanted format dd-mm-yy
